dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -114,6 +115,16 @@ protected Category findByKey(String key) throws DotDataException {
 		return cat;
 	}
 
+	@Override
+	protected Category findByVar(final String variable) throws DotDataException {

```

Comment:
I know the Category is still using Hibernate but maybe every new method should use dotConnect, to start the migration

Reply:
I would do it altogether later since this requires writing a new transformer etc